src/rdd/dbfcdx/dbfcdx1.c

```diff
@@ -54,8 +54,8 @@
 #  define HB_CDX_PACKTRAIL
 #endif
 
-#define HB_CDX_DBGCODE
 /*
+#define HB_CDX_DBGCODE
```

Comment:
Why this change? It seems to be totally unrelated to the socket part. IMHO needs a separate PR and some explanation

Reply:
It is a debug define for RARE situation with problem to CDX, if you check the line on prg dbfcdx1.c contains other defined already commented.